ac/ac-plot/src/index.js

```diff
@@ -33,32 +33,36 @@ const config = {
 
 const dataStructure = {};
 
-const mergeFunction = ({ data: incoming }, dataFn, data) => {
-  if (Array.isArray(incoming)) {
-    incoming.forEach(item => mergeFunction({ data: item }, dataFn, data));
-    return;
-  }
-  if (isEmpty(incoming) || !isObject(incoming)) {
-    return;
-  }
-  if (incoming['1']) {
```

Comment:
This is weird, did you write this special case to accept incomingData with the format:
`{  '1': { x, y, trace } }` ?
That looks arbitrary and unnatural to me. 
Made it so that the accepted types are only  `{x, y, trace}` or  `Array<{x, y, trace}>`